include/kernel-build.mk

```diff
@@ -138,8 +138,9 @@ define BuildKernel
   $(LINUX_DIR)/.modules: $(STAMP_CONFIGURED) $(LINUX_DIR)/.config FORCE
 	$(Kernel/CompileModules)
 	touch $$@
+	$(if $(CONFIG_STRIP_KERNEL_EXPORTS),$($(KERNEL_BUILD_DIR)/symtab.h))
```

Comment:
This line makes no sense to me. What do you think it does?

Reply:
The original line was added in bb5a40c64b7c4f4848509fa0a6625055fc9e66cc as part of the optional flag `CONFIG_STRIP_KERNEL_EXPORTS` to reduce the size of the kernel of unnecessary export symbols.
But for some reasons this breaks compilation on Raspberry PI3 64-bit when enabled (see bug 1282) because `vdso` on `arm64` complains about this missing header.
Moving this build up step up seems to solve it.

Do you think this could be solved in a better way?

Reply:
You seem to have confused make target context vs command context.
The line you added above does nothing, because $($(KERNEL_BUILD_DIR)/symtab.h)) resolves to an empty string
You probably intended to add it to the $(LINUX_DIR)/.modules: line. Unfortunately that is also broken, because the symtab.h needs to be generated after all packages have been built.
So I guess you need to create a dummy symtab.h earlier and leave the dependency for the final one where it is now